frontend/ui/src/pages/Inbox/Messenger/MessengerContainer/index.tsx

```diff
@@ -35,10 +35,10 @@ const MessengerContainer = ({
   const [suggestions, showSuggestedReplies] = useState<Suggestions>(null);
 
   useEffect(() => {
-    if (!currentConversation) {
+    if (!currentConversation && match.params.conversationId) {
       getConversationInfo(match.params.conversationId);
     }
-  }, [conversations]);
+  }, [currentConversation, match.params.conversationId]);
 
   const hideSuggestedReplies = () => {
```

Comment:
there was a 404 loop when opening inbox because the endpoint was called  even if there was no conversation ID